lucien-ios/Add New Comic Book/AddComicViewController.xib

```diff
@@ -0,0 +1,372 @@
+                                        <constraint firstAttribute="trailing" secondItem="uhS-Ot-Cen" secondAttribute="trailing" id="Xtf-ur-zG2"/>
+                                        <constraint firstItem="rah-tt-duF" firstAttribute="leading" secondItem="qSd-Yt-2xk" secondAttribute="leading" id="sPW-PM-XZZ"/>
+                                    </constraints>
+                                </stackView>
+                                <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" alignment="top" spacing="12" translatesAutoresizingMaskIntoConstraints="NO" id="eS6-d2-1CF">
+                                    <rect key="frame" x="0.0" y="735.5" width="320" height="78"/>
+                                    <subviews>
+                                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="RELEASE DATE" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="oyx-gl-cmU">
+                                            <rect key="frame" x="0.0" y="0.0" width="116.5" height="21"/>
+                                            <constraints>
+                                                <constraint firstAttribute="height" constant="21" id="3eT-Dk-KUW"/>
+                                            </constraints>
+                                            <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                            <nil key="textColor"/>
+                                            <nil key="highlightedColor"/>
+                                        </label>
+                                        <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="UOn-Nc-ucF">
+                                            <rect key="frame" x="0.0" y="33" width="320" height="32"/>
+                                            <fontDescription key="fontDescription" type="system" pointSize="16"/>
+                                            <state key="normal" title="Select a Date"/>
+                                            <connections>
+                                                <action selector="selectADateButtonTapped:" destination="-1" eventType="touchUpInside" id="4r4-bM-0jR"/>
+                                            </connections>
+                                        </button>
+                                        <datePicker hidden="YES" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" datePickerMode="dateAndTime" minuteInterval="1" translatesAutoresizingMaskIntoConstraints="NO" id="Ayn-n0-Cph">
```

Comment:
Your datePicker probably shouldn't include time as well. That might be useful for the due date but I don't think release time is something that's known